DependencyInjection/TacticianExtension.php

```diff
@@ -21,7 +21,6 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
         $loader->load('services.yml');
 
         $this->configureCommandBuses($mergedConfig, $container);
-        $this->injectMethodNameInflector($mergedConfig, $container);
```

Comment:
Since all handler locators are now created in the compiler pass, this functionality has moved there